tango/common/registrable.py

```diff
@@ -340,3 +347,14 @@ def _get_suggestion(name: str, available: List[str]) -> Optional[str]:
         if suggestion in available:
             return suggestion
     return None
+
+
+def _fullname(c: type) -> str:
+    return f"{c.__module__}.{c.__qualname__}"
```

Comment:
Is this about making pickle work?

Reply:
No, I just pulled this function out to the module level (previously it was just a locally defined function in `.register()`.